source/software/debuggers/index.rst

```diff
@@ -0,0 +1,281 @@
+
+        1. Click Accept.
+
+        2. Choose any appropriate options, and click Run.
+
+        The debugging interface opens. 
+
+        - Delayed command responses (ex: "ls", "mv") from tfe02 -
+          bastion-hera.princeton
+        - Downloading data to Jet front-end nodes (fe1) using curl
+          hangs/timesout
+
+    .. tab-item:: Hera or Jet
+
+        .. note::
+            Since DDT is GUI debugger, interactions over a wide area network can be extremely slow. You may want to consider using a Remote Desktop which in our environment is X2GO
+
+        For debugging you will need interactive access to the desired set of compute nodes using salloc with the desired set of resources:
+
+        .. code-block:: shell
+            $ salloc --x11=first -N 2 --ntasks=4 -A <project> -t 300 -q batch
+
+        At this point you are on a compute node.
+
+        **Load the desired modules**
+
+        .. code-block:: shell
+            $ module load intel impi forge
+
+        **Launch the application with the debugger**
+
+        .. code-block:: shell
+            % ddt srun -n 4 ./hello_mpi_c-intel-impi-debug
+
+        This will open GUI in which you can do your debugging. Please note that by default it seems to save your current state (breakpoints, etc. are saved for your next debugging session). 
+
+
+
+Launch Jobs Directly from DDT
+-----------------------------
+
+After loading all of the necessary modules, launch ddt GUI from a login node on preferred system. 
+
+.. code-block:: shell
+    $ ddt &
+
+
+.. note::
+    If using on Gaea, from the Remote Launch dropdown, click on the gaea configuration you had set up so your local
+    client connects to Gaea.
+
+
+Once the main window opnes, click on **Run** which should open up a window that looks something like this:
+
+.. figure:: /images/RUNmenu.png
+
+**Application**
+^^^^^^^^^^^^^^
+  
+This lets you specify the path to your application that you want to run.
+
+Click on the 'Details' button on the left to expand this section. Once expanded, you will be able
+to enter:
+
+**Application**: the path to the application
+
+**Arguments**: the args to pass to the application
+
+**stdin file**: any input file needed for the application
+
+**Working Directory**: the working directory from which the job should be started.
+
+**MPI**
+^^^^^^^
+
+Check the MPI box to indicate if your application is using MPI. Click on Details to expand this
+section.
+
+By default, you will only see the Implementation option which will say 'no MPI'. Click on the
+'Change..' button and a new window will open
+
+.. figure:: /images/ddtMPISettings.png
+
+From the MPI/UPC Implemenation dropdown, select 'Slurm (generic)' then click on OK to close
+the window.
+
+You will now have the option to set the 'Number of Processes' and 'Number of Nodes' to launch
+for your application.
+
+The 'srun arguments' section allow you to pass any additional arguments to th srun command.
+
+**Submit to Queue**
+^^^^^^^^^^^^^^^
```

Comment:
Remove either the bold or the underline.

Underline is too short.  It needs to be at least as long as the header text.

Reply:
fixed